package.json

```
@@ -9,7 +9,7 @@
   "scripts": {
     "lint": "eslint ./*.js src/*.js test/*.js",
     "test": "yarn push && cd test && node mbSheetsAddOn.js",
-    "push": "yarn fileversion && clasp push",
+    "push": "yarn fileversion && output=\"$(clasp push 2>&1)\"; if [ -n \"$(echo \"{$output}\" | grep 'code: 403')\" ]; then echo \"{$output}\"; exit 1; fi",
```

```suggestion
    "push": "yarn fileversion && output=\"$(clasp push 2>&1)\"; if [ -n \"$(echo \"{$output}\" | grep 'code: 403')\" ]; then echo \"{$output}\"; exit 1; else echo \"{$output}\"; fi",
```

Let's still output the results of `clasp push` in the success case, since I think it's useful to see what's going on:

```
$ yarn push
yarn run v1.22.10
$ yarn fileversion && output="$(clasp push 2>&1)"; if [ -n "$(echo "{$output}" | grep 'code: 403')" ]; then echo "{$output}"; exit 1; else echo "{$output}"; fi
$ echo "const VERSION = '$npm_package_version';" > src/Version.js
{(node:46610) ExperimentalWarning: The fs.promises API is experimental
└─ appsscript.json
└─ src/Code.js
└─ src/Code.spec.js
└─ src/Version.js
└─ src/library/Build.js
└─ src/library/Property.js
└─ src/library/Query.js
└─ src/library/Util.js
└─ src/library/Validate.js
Pushed 9 files.}
```